src/js/edu-benefits/components/SubmitMessage.jsx

```diff
@@ -63,11 +59,20 @@ export default class SubmitMessage extends React.Component {
               <span>{this.props.claimedBenefits}</span>
             </li>
             <li>
-              <b>Your relinquished benefits</b><br/>
+              <b>Benefits you gave up</b><br/>
               <span>{this.props.relinquishedBenefits}</span>
             </li>
           </ul>
         </div>
+        <h4>Documents</h4>
+        <p>This form doesn’t currently allow you to upload documents. Nothing is required right now. VA will review your application.</p>
+        <p><b>VA may eventually need the following documents:</b></p>
+        <ul>
+          <li></li>
```

Comment:
Perhaps bad form, but those were left blank because we need to populate that list based on how people fill out the form. If they select kicker, we can display a message about their kicker document. Same goes for if they select that they have made contributions of $600 or less. We have placeholder language for that in the wireframes, but rewrites of that were not included in the content we were given. Best if I put the placeholder language in there for now?